eas.json

```diff
@@ -20,6 +28,40 @@
       "android": {
         "buildType": "apk"
       }
+    },
+    "test": {
+      "extends": "base",
+      "android": {
+        "gradleCommand": ":app:assembleRelease :app:assembleAndroidTest -DtestBuildType=release",
+        "withoutCredentials": true
+      },
+      "ios": {
+        "simulator": true
+      },
+      "env": {
+        "EXPO_PUBLIC_DETOX_MOCKED": "mocked",
+        "EXPO_PUBLIC_AUTH_HOST": "https://auth.stage.graasp.org",
+        "EXPO_PUBLIC_API_HOST": "https://api.stage.graasp.org"
```

Comment:
I think this is equivalent as using an env file. Is there a way to use `.env.test`?

Reply:
FOR LOCAL TEST: I created an `.env.test` file, so every time we want to run the tests in local you should run the server with `NODE_ENV=test yarn start --dev-client`. So now we have a `.env.development` and `.env.test` files.

FOR EAS TESTS: These lines are only used in the EAS service, they are executed when you create a new build using EAS with `eas build --profile test`, so there are 3 options:
- Factor out these lines in another file
- Use the .env.test file but we need to run the command as `NODE_ENV=test eas build --profile test` to load the envs correctly from the test environment
- Leave it as it is now

Updated README.